packages/eslint-plugin/src/stylex-no-unused.js

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+'use strict';
+
+/*:: import { Rule } from 'eslint'; */
+import type {
+  CallExpression,
+  Expression,
+  Node,
+  Program,
+  Property,
+  SpreadElement,
+  RestElement,
+  MemberExpression,
+  AssignmentProperty,
+  ExportDefaultDeclaration,
+} from 'estree';
+
+type PropertyValue =
+  | Property
+  | SpreadElement
+  | AssignmentProperty
+  | RestElement;
+function isStylexCallee(node: Node) {
```

Comment:
Rename the function name:

```suggestion
function isStylexCreate(node: Node) {
```

Also, you will need to expand this to support various patterns for imports.

Internally, all our code uses the name `stylex` for the import, but in OSS, you can do things such as:

```ts
import * as s from '@stylexjs/stylex'
const styles = s.create({...})
```

```ts
import {create} from '@stylexjs/stylex'
const styles = create({...})
```

```ts
import {create as c} from '@stylexjs/stylex'
const styles = c({...})
```

All of these patterns will need to be supported.

Reply:
This is a great call, did not realize these cases are valid. Added them as additional functionalities of the feature.